src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -0,0 +1,255 @@
+            }
+
+            @Override
+            protected boolean isFinished() {
+                // Make this return true when this Command no longer needs to run execute()
+                if (leftEncoder.get() + rightEncoder.get() / 2 > distance)
+                    return true;
+                else
+                    return false;
+            }
+
+            @Override
+            protected void end() {
+                gyroDrivePID.disable();
+            }
+        };
+    }
+
+    public Command turnXDegrees(double degrees) {
+        // The command is named "Sample Command" and requires this subsystem.
+        return new Command("turnXDegrees", this) {
+            @Override
+            protected void initialize() {
+                gyroDrivePID.reset();
+                gyroDrivePID.setSetpoint(degrees);
+                gyroDrivePID.enable();
+            }
+
+            @Override
+            protected void execute() {
+                drive.arcadeDrive(0, gyroCorrection);
+            }
+
+            @Override
+            protected boolean isFinished() {
+                // Make this return true when this Command no longer needs to run execute()
+                if (gyro.getAngle() == degrees)
+                    return true;
+                else
+                    return false;
+            }
+
+            @Override
+            protected void end() {
+                gyroDrivePID.disable();
+            }
+        };
+    }
+
+    public Command align(double degrees) {
+        // The command is named "Sample Command" and requires this subsystem.
+        return new Command("turnXDegrees", this) {
+            @Override
+            protected void initialize() {
+                gyroDrivePID.reset();
+                gyroDrivePID.setSetpoint(degrees);
+                gyroDrivePID.enable();
+            }
+
+            @Override
+            protected void execute() {
+                drive.arcadeDrive(0, gyroCorrection);
+            }
+
+            @Override
+            protected boolean isFinished() {
+                // Make this return true when this Command no longer needs to run execute()
+                if (gyro.getAngle() == degrees)
+                    return true;
+                else
+                    return false;
+            }
+
+            @Override
+            protected void end() {
+                gyroDrivePID.disable();
+            }
+        };
+    }
+
+    public Command extendPiston() {
+        return new InstantCommand("Extend Piston", this, () -> {
+            climbPiston.set(Value.kForward);
+        });
+    }
+
+    public Command retractPiston() {
+        return new InstantCommand("Retract Piston", this, () -> {
+            climbPiston.set(Value.kReverse);
+        });
+    }
+
+    public Command score() {
+        // The command is named "Sample Command" and requires this subsystem.
+        return new Command("Score", this) {
```

Comment:
1. Create command `score()`
2. Fill out functions
3. ???
4. Profit

What does this function do?

Reply:
·